src/main/webapp/app/entities/result.model.ts

```diff
@@ -17,6 +17,7 @@ export class Result implements BaseEntity {
     public rated: boolean;
     public hasComplaint: boolean;
     public exampleResult: boolean;
+    public isAssessedTwice: boolean | null;
```

Comment:
When can it be null? Does this make sense here?

Reply:
Yeah, for non-exam exercises this value is never set,